exasol_script_languages_container_ci_setup/lib/run_start_release_build.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+from typing import Tuple, Dict
+
+from exasol_script_languages_container_ci_setup.lib.aws_access import AwsAccess
+
+
+def get_environment_variable_override(env_variable: Tuple[str, str]) -> Dict[str, str]:
+    return {"name": env_variable[0], "value": env_variable[1], "type": "PLAINTEXT"}
+
+
+def run_start_release_build(aws_access: AwsAccess, project: str, upload_url: str, branch: str, dry_run: bool) -> None:
+    """
+    This command executes
+    1. Retrieve all AWS Codebuild projects for this account
+    2. Find the one which matches the given project
+    3. Creates the environment variables override
+    4. Start and wait for batch build
+    :raises:
+        `RuntimeError` if build goes wrong or if anything on AWS CodeBuild is not as expected
+        `ValueError` if project is not found on AWS CodeBuild.
+    """
+    logging.info(f"run_start_release_build for aws profile {aws_access.aws_profile} for project {project} "
+                 f"with upload url: {upload_url}")
+    projects = aws_access.get_all_codebuild_projects()
+    matching_project = [p for p in projects if p.startswith(f"{project}ReleaseCodeBuild")]
```

Comment:
Better approach here would be:
1. Get Cloudformation stack for project (this has unique name identified by project name)
2. Analyze all components of stack and find the one of type CodeBuild